fashion_mnist/fashion_mnist_solution.py

```diff
@@ -157,8 +158,8 @@
 
 # Train the model. The training will randomly sample 1 / N batches of training data and
 # 3 / N batches of validation data on every worker, where N is the number of workers.
-# Over-sampling of validation data helps to increase probability that every validation
-# example will be evaluated.
+# Over-sampling of validation data, which helps to increase the probability that every
```

Comment:
If this is an instruction or description, recommend Over-sampling --> This example configures oversampling

Reply:
I'll skip this because it would require re-taking screenshots.